commons/build.gradle

```diff
@@ -16,5 +16,5 @@ bootJar {
 }
 
 dependencies {
-    compile "com.fasterxml.jackson.core:jackson-databind:2.10.3"
+    compile "com.fasterxml.jackson.core:jackson-databind:2.11.1"
```

Comment:
not the best way of doing it, you're duplicating the configuration across multiple subprojects.

you should be able to force it at the top level with something like:
https://github.com/hmcts/ccd-user-profile-api/blob/RDM-10549-master/build.gradle#L436

and then not specifying the version anywhere else.

Still approving as it looks like it continues an existing pattern and it's better to have the build green, but would be a nice small improvement imo